src/leaguepedia_client.py

```
@@ -107,13 +107,11 @@ async def login(self):
 
             username = user_data.get("name")
             logger.info(
-                f"Successfully logged in to Leaguepedia as '{username}'."
+                "Successfully logged in to Leaguepedia as '%s'.", username
             )
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)